build.gradle

```diff
@@ -276,6 +276,7 @@ project(':cruise-control') {
     implementation "org.apache.zookeeper:zookeeper:${zookeeperVersion}"
     implementation "io.netty:netty-handler:${nettyVersion}"
     implementation "io.netty:netty-transport-native-epoll:${nettyVersion}"
+    implementation "software.amazon.msk:aws-msk-iam-auth:2.0.2"
```

Comment:
I think I need to use ' ' instead

Reply:
Why do you think you need single quotes? All other `implementation` lines are with double quotes.
I'm just curious about the rationale of your comment.
Also to follow the convention you should probably add something like "iamVersion" variable.